beets/library.py

```diff
@@ -1502,13 +1504,20 @@ def tmpl_aunique(self, keys=None, disam=None):
 
         else:
             # No disambiguator distinguished all fields.
-            res = u' {0}'.format(album.id)
+            res = u' {1}{0}{2}'.format(album.id, bracket[0] if bracket else
+                                       u'', bracket[1] if bracket else u'')
             self.lib._memotable[memokey] = res
             return res
 
         # Flatten disambiguation value into a string.
         disam_value = album.formatted(True).get(disambiguator)
-        res = u' [{0}]'.format(disam_value)
+        res = u' {1}{0}{2}'.format(disam_value, bracket[0] if bracket else u'',
+                                   bracket[1] if bracket else u'')
```

Comment:
Maybe this and the prior line with bracket-formatting would be more legible if the two brackets were assigned into variables?

```py
if bracket:
    bracket_l = bracket[0]
    bracket_r = bracket[1]
else:
    bracket_l = u''
    bracket_r = u''
```